### ISSUE TITLE
Continue does not increment counter of for loop

### ISSUE THREAD

**Issue by Tarnadas** [AssemblyScript/assemblyscript#150]
```js
for (let i = 0; i < 10; i++) {
  for (let j = 0; j < 10; j++) {
    if (i === j) continue; // won't increase j by 1, therefore this is an endless loop
  }
}
```

**Comment by Tarnadas** [comment]
fixed by 829af2285c75db6ba3a8b1b9766dcb2dfd8eefdc

**Comment by dcodeIO** [comment]
Related: https://github.com/AssemblyScript/assemblyscript/commit/0d64c9661a60446164c38bf414613ca8ac9cf88e
